### DISCUSSION TITLE
How do you manage multiple environments using Nuxt and VueFire?

### DISCUSSION THREAD

**Discussion by gabyshev** [vuejs/vuefire#1409] [Questions and Help] [answered]
Let's say I want to deploy same Nuxt app to staging before deploying to production.

I was planning to use runtime variables but that seems to not work. https://github.com/vuejs/vuefire/discussions/1363

How do you guys do that?

**Comment by posva** [comment, marked as answer]
You need to define them at build time as some of these are inline in the client bundle. I think the best way is to have multiple `.env` files [like suggested in Nuxt docs](https://nuxt.com/docs/guide/directory-structure/env#env-file)
